docs/markup-variables-blocks.md

```diff
@@ -3,4 +3,184 @@ id: markup-variables-blocks
 title: Variables and Blocks
 ---
 
-**TBD**
+Grammar
```

Comment:
Some of that text might be "refactored" earlier, e.g., in the `Overview` section.